### ISSUE TITLE
[Feature Request] Mission Condition assigngear factions.

### ISSUE THREAD

**Issue by Khaosmatic** [ferstaberinde/F3#674]
Currently the system in place means we can dictate a set of gear per faction, e.g. NATO, OPFOR and INDEP. By faction I mean you set up the assigngear to have US Marines gear, etc. Wouldn't it be feasible to be able to create a folder wherein it has for example:

'f_assignGear_faction1.sqf'
'f_assignGear_faction2.sqf'

And then in the mission conditions you could just have 'Use faction: 1, 2 or 3' just like weather or time? Given I don't have extensive knowledge in this but you are executing an .sqf for weather and time presets so surely this is possible? Also I guess it doesn't really apply as F3 is built for vanilla ARMA where there is only one OPFOR or INDEPENDANT faction however for people using mods it means replay-ability could be added to missions very easily.

**Comment by Pasrules** [comment]
For B|Tac we have 3 factions which we keep the same NATO, CSAT, FIA with their custom gear. Then we have multiple spare AAF loadouts which we just drag and drop into F3 if we want to use them. This could be made a param but alot of script would have to be changed over.

**Comment by Khaosmatic** [comment]
Indeed, I too have set up multiple custom gear profiles which can be dropped in, its just when PBO'd or on a dedicated server it means either you shut down and upload the new mission or bog down the server with multiple types.

**Comment by ferstaberinde** [comment]
This may constitute bloat, but am going to assign to the 'future' milestone for now (it will not be in scope for v3-4-0).

**Comment by RayGab** [comment]
Did you mean something like this....
// F3 - LoadoutSelector

// This Parameter will select the Loadouts that are chosen for each Side

	class f_param_uniforms
	{
			title = "$STR_f_param_uniforms";
			values[] = {0,1,2,3};
			texts[] = {"Vanilla","RHS Woodland","RHS Desert","CUP Woodland"};
			default = 1;
			code = "f_param_uniforms = %1";

If you want I can start a PR without the other changes we made.

**Comment by Khaosmatic** [comment]
Yes that is exactly it! I apologize I only received the notification for this today.
